### ISSUE TITLE
M1 support arraykit having to build from wheel

### ISSUE THREAD

**Issue by alexlatif** [static-frame/static-frame#783]
**Description**

Thanks Chris for your lib. Just what we needed. However, there's an arraykit issue not support M1/M2. So we have to uninstall arraykit and install non prebuilt wheels.

```bash
pip install --no-binary :all: arraykit==0.3.1 
```

You also mention arraykit>=0.3.2 but it is forcing 0.3.1 install though not the issue.

Python version > 3.11 and 3.10

**Comment by flexatone** [comment]
Many thanks for reporting this, @alexlatif . Let me double check the build configuration for arraykit and see what I can do. I am publishing "universal2" wheels (https://files.pythonhosted.org/packages/b1/18/6668ab86cb47616f491cf1b368548dedb0c0a9ee563b02af101701e72e43/arraykit-0.3.3-cp311-cp311-macosx_10_9_universal2.whl), which I thought covered M1/M2, but there might be more I can do.

I will get some new release out in a few days.

**Comment by flexatone** [comment]
@alexlatif : can you let me know what version of NumPy you are using? I wonder if that is why you were forced into using 0.3.1. Also, if you have the error, please include it here.

**Comment by alexlatif** [comment]
v1.24.2 and i also just upgraded to the latest stable version v1.24.3. either way it forces arraykit==0.3.1

But irrespective of forcing this version when i install the latest arraykit version too I get the same error. 

My stacktrace:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/alessandrolatif/watz/venv/lib/python3.11/site-packages/static_frame/__init__.py", line 6, in <module>
    from arraykit import isna_element as isna_element
  File "/Users/alessandrolatif/watz/venv/lib/python3.11/site-packages/arraykit/__init__.py", line 5, in <module>
    from ._arraykit import __version__
ImportError: dlopen(/Users/alessandrolatif/watz/venv/lib/python3.11/site-packages/arraykit/_arraykit.cpython-311-darwin.so, 0x0002): symbol not found in flat namespace '_npy_double_to_half'
```

**Comment by alexlatif** [comment]
to get round this issue rn I uninstall arrakit after pip installing static-frame and am reinstalling the same forced version using the --no-binary flag which is soon to be deprecated pip functionality.

**Comment by flexatone** [comment]
This has been resolved with the 2.1.1 release and associated dependencies.
